doc/releases/nautilus.rst

```diff
@@ -1,3 +1,40 @@
+v14.2.14 Nautilus
+=================
+
+This is the 14th backport release in the Nautilus series. This releases fixes a
+security flaw affecting Messenger V2 for Octopus & Nautilus, among other fixes
+across components. We recommend users to update to this release.
+
+Notable Changes
+---------------
+
+* CVE 2020-25660: Fix a regression in Messenger V2 replay attacks
```

Comment:
"CVE 2020-25660: CEPHX_V2 replay attack protection lost (only for Messenger v2)"